pages/common/vifm.md

```diff
@@ -0,0 +1,17 @@
+# vifm
+
+> Vifm (VI File Manager) is a command line file manager.
+> For icons, themes, file previews and more, check out <https://github.com/vifm/vifm#other-resources>.
+> Check out the cheatsheet at <https://vifm.info/cheatsheets.shtml>.
```

Comment:
I do not think we include this information in the examples.
The more information link should be a good starting point :)
/cc @EmilySeville7cfg @navarroaxel Thoughts ?
```suggestion
```

Reply:
This program is filled with shortcuts having a cheatsheet is a very helpful way to get started with some commands. i was thinking about putting in some common keyboard shortcuts but it would be very bloated, this is why i left a cheatsheet for quick reference. The program is not usable without these important shortcuts.